src/wdio/services/SeleniumDockerService.js

```diff
@@ -0,0 +1,121 @@
+import { exec } from 'child_process';
+import retry from 'async/retry';
+import http from 'http';
+import path from 'path';
+import fs from 'fs';
+
+/**
+* Webdriver.io SeleniuMDockerService
+* provides standalone chrome/firefox selenium docker automation.
+*/
+export default class SeleniumDockerService {
+  constructor() {
+    this.getSeleniumStatus = this.getSeleniumStatus.bind(this);
+  }
+
+  getSeleniumStatus(callback) {
+    http.get({
+      host: this.host,
+      port: this.port,
+      path: path.join(this.path || '/wd/hub', 'status'),
```

Comment:
thinking we need to split '/wd/hub/' out to 'wd', 'hub'

Reply:
that doesn't really provide any value being `this.path` will be a pre-assembled path as well. Its really the path to the selenium hub endpoint wihch shouldn't change much. I just need to be able to reliably append `status` to it and not have to worry about a double //